contribs/freight/src/main/java/org/matsim/contrib/freight/jsprit/NetworkRouter.java

```diff
@@ -31,8 +31,8 @@
 	 * @param {@link CarrierPlan}
 	 * @param {@link NetworkBasedTransportCosts}
 	 */
-	public static void routePlan(CarrierPlan plan, NetworkBasedTransportCosts netbasedTransportCosts){
-		new TimeAndSpacePlanRouter(netbasedTransportCosts.getRouter(), netbasedTransportCosts.getNetwork(), netbasedTransportCosts.getTravelTime()).run(plan);
+	public static void routePlan(CarrierPlan plan, TransportCosts freightTransportCosts){
+		new TimeAndSpacePlanRouter(freightTransportCosts.getRouter(), freightTransportCosts.getNetwork(), freightTransportCosts.getTravelTime()).run(plan);
```

Comment:
This is currently the only place where the (package-private) `.getRouter()` is used.
`getNetwork() `and `getTravelTime()` could at least inside MATSim be package-private as well since this here is also the signle location, where it is used. but I do not know if someone outside will used it.